graphql/database/users/createUser.js

```diff
@@ -51,6 +55,14 @@ const createUser = async (
       ? {
           extensionInstallTimeApprox,
         }
+      : null,
+    v4BetaEnabled
+      ? {
+          backgroundImage: await getRandomBackgroundImage(
```

Comment:
Like you suggested, you will have to map the background image to modify it a little. [Here's an example](https://github.com/gladly-team/tab/blob/master/graphql/database/users/setBackgroundImageDaily.js#L22) where we do that elsewhere. The timestamp field is when the background was last set, so you can use a current ISO timestamp.